lighthouse-core/audits/byte-efficiency/uses-responsive-images-snapshot.js

```diff
@@ -17,6 +17,10 @@ const URL = require('../../lib/url-shim.js');
 const i18n = require('../../lib/i18n/i18n.js');
 
 const UIStrings = {
+  /** Descriptive title of a Lighthouse audit that checks if images match their displayed dimensions. This is displayed when the audit is passing. */
+  title: 'Images were appropriate for their displayed size',
```

Comment:
It's not ideal to duplicate the title, but it's the best way to get around this new check. Could we do a quick unit test to enforce these titles are the same?

Reply:
Oh they're explicitly *not* the same because one is an opportunity which uses imperative titles and this one is a descriptive title that has the binary flip.